Documentation/helm-operator.md

```diff
@@ -88,40 +88,57 @@ The command removes all the Kubernetes components associated with the chart and
+| `csi.cephfsGrpcMetricsPort`     | CSI CephFS driver GRPC metrics port.                                                                    | `9091`                                                 |
+| `csi.cephfsLivenessMetricsPort` | CSI CephFS driver metrics port.                                                                         | `9081`                                                 |
+| `csi.rbdGrpcMetricsPort`        | Ceph CSI RBD driver GRPC metrics port.                                                                  | `9090`                                                 |
+| `csi.rbdLivenessMetricsPort`    | Ceph CSI RBD driver metrics port.                                                                       | `8080`                                                 |
+| `csi.kubeletDirPath`            | Kubelet root directory path (if the Kubelet uses a different path for the `--root-dir` flag)            | `/var/lib/kubelet`                                     |
+| `csi.cephcsi.image`             | Ceph CSI image.                                                                                         | `quay.io/cephcsi/cephcsi:v1.2.1`                       |
+| `csi.registrar.image`           | Kubernetes CSI registrar image.                                                                         | `quay.io/k8scsi/csi-node-driver-registrar:v1.1.0`      |
+| `csi.provisioner.image`         | Kubernetes CSI provisioner image.                                                                       | `quay.io/k8scsi/csi-provisioner:v1.3.0`                |
+| `csi.snapshotter.image`         | Kubernetes CSI snapshotter image.                                                                       | `quay.io/k8scsi/csi-snapshotter:v1.2.0`                |
+| `csi.attacher.image`            | Kubernetes CSI Attacher image.                                                                          | `quay.io/k8scsi/csi-attacher:v1.2.0`                   |
+| `discover.toleration`           | Toleration for the discover pods                                                                        | <none>                                                 |
+| `discover.tolerationKey`        | The specific key of the taint to tolerate                                                               | <none>                                                 |
+| `discover.tolerations`          | Array of tolerations in YAML format which will be added to discover deployment                          | <none>                                                 |
+| `discover.nodeAffinity`         | The node labels for affinity of `discover-agent` (***)                                                  | <none>                                                 |
+| `mon.healthCheckInterval`       | The frequency for the operator to check the mon health                                                  | `45s`                                                  |
```

Comment:
Could we reorder the table to have more commonly used settings higher? For example, move the mon settings up, then csi, then discover, then agent?

Reply:
I have changed the order to the top level values, mon, discover, csi and then agent.
I have moved the `discover` in front of the CSI as that small block might be lost otherwise and due to the importance having the discover Pod to even get OSDs running to be able to mount storage. :wink: